orchestrator/api.js

```diff
@@ -75,7 +78,7 @@ app.delete('/v1/node/:id', (req, res) => {
   nodeManager.delRemoteNode(req.params.id, req.service).then(() => {
     return res.status(200).send({message: 'ok'});
   }).catch((error) => {
-    console.log(error)
+    logger.warn(error, { filename: 'api' });
```

Comment:
Hi @raulnegreiros, very good! But maybe it would be better to use logger.error inside cacth?